server/package.json

```diff
@@ -22,7 +22,8 @@
     "test:e2e": "jest --config ./test/jest-e2e.json",
     "prisma:generate": "npx prisma generate",
     "prisma:generate-watch": "npx prisma generate --watch",
-    "prisma:studio": "npx prisma studio"
+    "prisma:studio": "npx prisma studio",
+    "prisma:db-push": "npx prisma db push"
```

Comment:
For syncing mongodb with `@unique` attributes 

https://github.com/prisma/prisma/releases/tag/3.4.0

Reply:
I won't approve this PR until the clock strikes 1pm